wunderlist-create-subtask.html

```diff
@@ -0,0 +1,46 @@
+<script type="text/javascript">
+  RED.nodes.registerType('wunderlist-create-subtask',{
+    category: 'function',
+    color: '#E84228',
+    defaults: {
+      config: { type: 'wunderlist-config', required: true },
+      subtaskTitle: { value: '' },
+      taskId: { value: '' },
+      name: { value: '' }
+    },
+    inputs:1,
+    outputs:1,
+    icon: 'wunderlist-logo.png',
+    label: function() {
+      return this.name || this.subtaskTitle || 'create Wunderlist subtask';
+    }
+  });
+</script>
+
+<script type="text/x-red" data-template-name="wunderlist-create-subtask">
+  <div class="form-row">
+    <label for="node-input-config"><i class="fa fa-code"></i> Configuration</label>
+    <input type="text" id="node-input-config" placeholder="Configuration">
+  </div>
+  <div class="form-row">
+    <label for="node-input-subtaskTitle"><i class="fa fa-thumb-tack"></i> Subtask Title</label>
+    <input type="text" id="node-input-subtaskTitle" placeholder="Milk">
+  </div>
+  <div class="form-row">
+    <label for="node-input-listId"><i class="fa fa-list-alt"></i> Task Id</label>
```

Comment:
`<label for "node-input-listId">` should be `<label for "node-input-taskId">`

Reply:
Ok, I will correct